lib/Models/CsvCatalogItem.js

```diff
@@ -419,14 +425,19 @@ function loadTableFromCsv(item, csvString) {
     tableStructure.loadFromCsv(csvString);
     setActiveTimeColumn(tableStructure, tableStyle);
     item._tableStructure = tableStructure;
+    // Does the csv have addresses we can translate to long and lat?
+    if (!tableStructure.hasLatitudeAndLongitude && tableStructure.hasAddress)
+    {
+        var addrConverter = new AddressConverter(tableStructure, item.terria.corsProxy);
+        return addrConverter.convertEachAddress().then(function() {
+            return createDataSourceForLatLong(item, tableStructure);
+        })
+        .otherwise(function(e) {
+            console.log("Unable to map addresses to lat-long coordinates.", e);
```

Comment:
This otherwise will prevent the error from being raised up higher and being shown to the user.  Either remove it, or add a `throw e` to the end.